lib/libvfio-user.c

```diff
@@ -1454,15 +1513,48 @@ vfu_reset_ctx(vfu_ctx_t *vfu_ctx, const char *reason)
     }
 }
 
+static int
+vfu_reset_ctx(vfu_ctx_t *vfu_ctx, int reason)
+{
+    if (vfu_ctx->quiesce != NULL
+        && vfu_ctx->pending.state == VFU_CTX_PENDING_NONE) {
+        int ret = vfu_ctx->quiesce(vfu_ctx);
+        if (ret < 0) {
+            if (errno == EBUSY) {
+                vfu_ctx->pending.state = VFU_CTX_PENDING_CTX_RESET;
+                vfu_ctx->pending.ctx_reset_errno = reason;
```

Comment:
seems a bit silly to stash the reason, *solely* to have something to print out on :1497 - can we print it out here instead?

Also,

Reply:
Sure we can print it here.